docs/docs/modeling.md

```diff
@@ -118,13 +118,13 @@ classDiagram
 
 TenantsWork refers to the set of tasks assigned(AssignedCategory) to each tenant.
 
-### Tenant IDs
+### Tenant ID
 
-The TenantsWork contains Tenants. The TenantsWork class has Tenant ID.
+The TenantsWork class has Tenant ID. TenantsWork will be created per Tenant.
```

Comment:
IMO, the description, `TenantsWork will be created per Tenant.` is not for the `Tenant ID`.